### DISCUSSION TITLE
EIP-4844 Blob Transactions

### DISCUSSION THREAD

**Discussion by serinuntius** [wevm/viem#1755] [Idea / Feature Request] [answered]
Hello viem devs,

I propose integrating EIP-4844 (Blob Transactions) into viem. This Ethereum Improvement Proposal offers significant benefits:

1. Reduced gas costs for large transactions.
1. Improved scalability with increased data throughput.
1. Enhanced efficiency and user experience.

Reference: [EIP-4844](https://eips.ethereum.org/EIPS/eip-4844)

Your consideration of this feature would greatly benefit our users dealing with high-volume transactions.

Thank you.

**Comment by jxom** [comment, marked as answer]
Support for EIP-4844 transaction formatters and types has been [merged](#1754).

Support for signing & sending EIP-4844 transactions is [underway](https://github.com/wevm/viem/pull/1740).

**Reply by serinuntius** [reply]
Thanks, I was searching for Issues and didn't see the PR.

Looking forward to it.
